src/assets/scripts/client/InputController.js

```diff
@@ -699,6 +699,17 @@ export default class InputController {
 
                 return true;
             }
+            case PARSED_COMMAND_NAME.AIRAC: {
+                const airacCycle = AirportController.getAiracCycle();
+
+                if (airacCycle === null) {
+                    UiController.ui_log('unknown airac cycle');
+                } else {
+                    UiController.ui_log(`airac cycle ${airacCycle}`);
```

Comment:
AIRAC is an acronym that (except in variable names and such) should be capitalized. Stands for Aeronautical Information Regulation And Control.

Reply:
done